zeebe/engine/src/test/java/io/camunda/zeebe/engine/processing/group/CreateGroupMultiPartitionTest.java

```diff
@@ -100,16 +99,9 @@ public void shouldDistributeInIdentityQueue() {
   public void distributionShouldNotOvertakeOtherCommandsInSameQueue() {
     // given the user creation distribution is intercepted
     for (int partitionId = 2; partitionId <= PARTITION_COUNT; partitionId++) {
-      interceptUserCreateForPartition(partitionId);
+      interceptGroupCommandForPartition(partitionId, GroupIntent.CREATE);
```

Comment:
❌ You're not testing if commands don't overtake each other here. You're only sending one command, so all that's being tested is the redistribution of that command. To test that stuff doesn't overtake each other you'll have to send another distributed command on the same queue, with a different value type or intent. Then you can guarantee it's ordered.

Reply:
@nicpuppa I'm creating another group on line 108, so two `GroupIntent.CREATE` commands are sent.

Or did I misunderstand something?